hydrus/app_factory.py

```diff
@@ -20,6 +21,13 @@ def app_factory(api_name: str = "api", vocab_route: str = "vocab") -> Flask:
     app.url_map.strict_slashes = False
     api = Api(app)
 
+
+    @app.after_request
+    def logging_request_response(response):
+        logger.info(" {} {}:{} Object_created_at : {}, status_code : {} "
```

Comment:
I think this line should be added inside an if-else block ( `if response.location : `) to check if we're getting a `response.location` or not. Otherwise `Object_created_at :` will keep returning `None` even in case of GET requests to other endpoints. In that case, we should only log the method and path of request.

Reply:
@farazkhanfk7 I have done that, just waiting for the level `DEBUG` approval to push all the changes.
Thanks.

Reply:
please see my comment below

Reply:
@Mec-iS yes, I am figuring out the way to deal with those logs with structlog.